hs_core/templates/pages/genericresource.html

```diff
@@ -498,14 +516,20 @@ <h2 id="resource-title">{{ title }}</h2>
                         </div>
                     {% endif %}
                 {% else %}
-                    {% if cm.raccess.public %}
+                    {% if cm.raccess.published %}
+                        <strong class="label-public">&nbspPublished</strong>
```

Comment:
@selimnairb I followed the rest of sharing status display format and they all used ```$nbsp``` rather than ```$nbsp;``` such as showing Public, Discoverable, or Private, and they all seem to work correctly. @mauriel133 is there is reason to use ```$nbsp``` rather than ```$nbsp;``` here? I am afraid something will be broken when displaying the page on a mobile device for example if I change ```$nbsp``` to ```$nbsp;``` @mauriel133 Please let me know whether it is safe to change ```$nbsp``` to ```$nbsp;``` or just leave them as is.